### ISSUE TITLE
Find alternative or a way to include Font Awesome (FA) in Angular external HTML files

### ISSUE THREAD

**Issue by bytech-ca** [eclectic-bytech/webcourses-ngx-v4#248]
We can attach CSS and JS files to Angular, and that might be one way to allow FA in external HTML pages.

Another solution might be to go through the source code on a FA Angular icon, and see what it compiled to (by using web dev tools to see what code it shows.)

**Comment by SirL3X0N** [comment]
doing this instead of just using images seems like a royal waste of time but I'll get on it

**Comment by bytech-ca** [comment]
Some reasons for a non-programmatic implementation include

- the option to use icons inside web courses themselves
- using images doesn't offer a standardised user experience, which makes for a poor UX and is one of the reasons packages like font awesome were developed
- increase platform user-friendliness by saving admins the need to know how to source, scale, secure rights and make responsive every iconic image they might want to apply

**Comment by SirL3X0N** [comment]
> 

Yes I did but it should've been obvious from the start as we are calling it on line 50 already

**Comment by bytech-ca** [comment]
Sorry, what should be obvious?

**Comment by SirL3X0N** [comment]
> Sorry, what should be obvious?

the fact that calling it won't work

**Comment by SirL3X0N** [comment]
> 

We were already importing what you wanted me to try out. It doesn't work there or in the other style section

**Comment by bytech-ca** [comment]
The angular.json import works, but the css file is for FontAwesome 4.7 which does not contain the icons we are using on the contact page.

When I copied the CDN file for FA 6.4 and imported it in angular.json like suggested above, it threw a fit. When loading from the CDN, it imports on the CDN side other files missing in the node_modules font_awesome directory.

Since the font-awesome npm module does not have a version newer than 4.7, it does seem like the best way to add the option for FA6.x icons is the one you already discovered: by doing the import using `link` in index.html. I've implemented that solution on the 'move contact page out of code into publisher-files' branch. Let's take a look at it together during the next teleconference, before applying it to the other files that need their content externalised.
